conjure-core/src/main/java/com/palantir/conjure/defs/ConjureParserUtils.java

```diff
@@ -393,7 +403,7 @@ static List<FieldDefinition> parseField(
                 .collect(Collectors.toList());
     }
 
-    private static LogSafety parseLogSafety(LogSafetyDefinition def) {
+    public static LogSafety parseLogSafety(LogSafetyDefinition def) {
```

Comment:
Are there any uses of this method outside this class? We may be able to keep it private

Reply:
It's used in ConjureTypeParserVisitor, but I can make it package-private

Reply:
oh yes, you're totally right 🤦 fixing